styles/ServicesTitle.module.css

```diff
@@ -0,0 +1,17 @@
+.service_title {
+  padding-bottom: 63px;
+  padding-top: 63px;
+  border-bottom: 1px solid var(--border-default);
+}
+.service_title__heading {
+  padding-top: 16px;
+  font-weight: 500;
+}
+
+.service_title__button {
+  width: 22%;
```

Comment:
I think we need not define width for the button, as its width will adapt to the size of the text

Reply:
This width is not for buttons, it should work on the container with four buttons.  I have changed the className, maybe more specific